package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "next build",
     "start": "next dev",
-    "lint": "next lint",
```

Comment:
keep the lint command for ci to ensure the pr passes linting
add additional lint:fix for better dev exp to lint before committing. alternatively add git commit hooks to do lint precommit. (nice to have)

Reply:
Are you worried about the CI failing because of lint fix?

Per our convention, lint errors are auto fixed when saving the file.

Reply:
i am not sure if i quite understand what your intention is. 
but based on the current pipeline it runs 
![image](https://github.com/OpenBCca/openbc-web/assets/8355092/aa59c9f8-2b83-48f6-a4ca-f629c710c39e)
..
and if the command is modified to next lint --fix, then it is not doing a linting check at all. it is doing a lint fix on the file. then throwing it away. it's almost as if you run the test, letting the test auto fixes itself. then throw away the change, then keeping the unfixed version.

the pipeline should do the lint check, not lint fix
if we want the dev to run lint fix locally. create another command such as lint:fix but keep the lint for the pipeline.

Reply:
`Per our convention, lint errors are auto fixed when saving the file.`
- local configuration or local preferences shouldn't be dependent on. the pipeline is the ultimate test in terms of code quality (whether it's linting, testing, code coverage).